pkg/master/master.go

```diff
@@ -226,6 +233,14 @@ func setDefaults(c *Config) {
 		}
 		c.PortalNet = portalNet
 	}
+	if c.NodePorts.Size == 0 {
+		// TODO: Currently no way to specify no NodePort range (do we need to allow this?)
+		// We should probably allow this for clouds that don't require NodePort to do load-balancing (GCE)
+		// but then that breaks the strict nestedness of visibility.
+		// Review post-v1
+		c.NodePorts = util.PortRange{Base: 30000, Size: 2767}
```

Comment:
define a variable defaultRange in this scope, to make it  a bit more obvious that these magic numbers are defaults.